src/frontends/pytorch/src/op/remainder.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_remainder(NodeContext& context) {
+    auto x = context.get_input(0);
```

Comment:
can you please add a check that the node has at least two inputs? You can take a loot at TF FE where we have such routine:)

Reply:
We discussed it with @mvafin, probably it is better to have common helper, so I make it as separated request for covering all ops